benchmarks/import_pystac.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+
+
+class ImportPySTACBench:
+    repeat = 10
+
+    def setup(self, *args: Any, **kwargs: Any) -> None:
+        def import_pystac() -> None:
+            import pystac  # noqa: F401
+
+        self._import_pystac = import_pystac
+
+    def time_import_pystac(self) -> None:
+        self._import_pystac()
```

Comment:
Per the asv docs, I think this should be using [raw timing](https://asv.readthedocs.io/en/stable/writing_benchmarks.html#raw-timing-benchmarks).

Reply:
Good catch, thanks.